tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,471 @@ def test_reads_during_server_pod_restart(self):
+                                                     f"{fail_bkt_get} {event_bkt_get}"
+                                                     f"or DI_CHECK: {fail_di_bkt} {event_di_bkt}")
+        LOGGER.info("Step 7: Successfully verified READs & DI check for remaining buckets: %s",
+                    buckets)
+        LOGGER.info("Step 8: Check READ/Verify on data written in healthy mode and delete buckets")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users_org.values())[0],
+                                                    log_prefix=self.test_prefix, skipwrite=True,
+                                                    setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 8: READ/Verify on data written in healthy mode was successful and "
+                    "buckets deleted.")
+        LOGGER.info("Step 9: Create IAM user with multiple buckets and run IOs when "
+                    "server pod %s is down.", pod_name)
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
+        self.test_prefix = 'test-44840-1'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    nsamples=2, nclients=2)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 9: Successfully created IAM user with multiple buckets and ran IOs.")
+        LOGGER.info("COMPLETED: Test to verify continuous DELETEs during server pod restart.")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-44838")
+    def test_read_write_during_server_pod_restart(self):
+        """
+        Verify READ/WRITEs during server pod restart
+        """
+        LOGGER.info("STARTED: Test to verify READ/WRITE during server pod restart.")
+        LOGGER.info("STEP 1: Perform WRITEs/READs/Verify with variable object sizes")
+        LOGGER.info("Create 2 set of buckets to be used for writes and reads")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
+        test_prefix_read = 'test-44838-read'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_read, skipcleanup=True,
+                                                    nsamples=5, nclients=5, skipread=True)
+        assert_utils.assert_true(resp[0], resp[1])
+        test_prefix_write = 'test-44838-write'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_write, skipcleanup=True,
+                                                    nsamples=5, nclients=2, setup_s3bench=False)
```

Comment:
any specific reason for nclients 2 here?

Reply:
No reason as i can see, should you want to change it to 5?

Reply:
yeah lets make it consistent for reads and writes